vendor/github.com/zorkian/go-datadog-api/datadog-accessors.go

```diff
@@ -10213,6 +10213,68 @@ func (m *Monitor) SetType(v string) {
 	m.Type = &v
 }
 
+// GetEnd returns the End field if non-nil, zero value otherwise.
```

Comment:
I guess we have so many accessors because you added all the one that were missing even tho there are not being used here, Is that it?

Reply:
This is vendored from the go-datadog-api, which has an automatic generator for accessors that is required to be run after any change/addition of a struct. So yeah, it's not used ATM, but the design of the library is such that it's there.